runtime-modules/forum/src/tests.rs

```diff
@@ -2588,3 +2588,50 @@ fn storage_limit_checks() {
         }
     });
 }
+
+#[test]
+fn add_non_editable_post_should_not_increase_thread_number_of_posts() {
+    pub const INITIAL_BALANCE: u64 = 10_000_000;
```

Comment:
Please make it `ed() + thread_deposit + post_deposit`, as those parameters may change